test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -19,8 +19,9 @@ function usage() {
     Push the latest Antrea image to all vagrant nodes and restart the Antrea daemons
           --prometheus                 Deploy Prometheus service to scrape metrics
                                        from Antrea Agents and Controllers.
-          --flow-collector <Addr|ELK>  Provide either the external IPFIX collector
-                                       address or specify 'ELK' to deploy the ELK
+          --flow-collector <Addr|ELK|clickhouse>
```

Comment:
Our new flow collector is called Grafana flow collector, so I think it would be better to change this argument from `clickhouse` to `Grafana`?